plugins/modules/firewall.py

```diff
@@ -153,6 +153,11 @@
         editable=True,
         description=["The status of this Firewall."],
     ),
+    "tags": SpecField(
+        type=FieldType.list,
+        editable=True,
+        description=["A list of tags to apply to this Firewall."]
+    ),
```

Comment:
This isn't directly related to this fix but it's a gap I noticed that shouldn't cause any functional changes 🙂